components/team-members.js

```diff
@@ -11,6 +11,12 @@ class TeamMembers extends Component {
         <style dangerouslySetInnerHTML={{ __html: stylesheet }} />
         {this.props.team.map(member => (
           <article key={member.name} className="Team__members--card">
+          {/*  {member.name !== 'Jose Aguinaga' &&
```

Comment:
Can you just delete this? No need to comment it, we have our source version control for that.